src/resolve-dependency.js

```diff
@@ -18,8 +21,8 @@ function resolvePath (path, parent, job) {
 }
 
 function resolveFile (path, parent, job) {
+  if (path[path.length - 1] === '/') return;
```

Comment:
I noticed you're using `/` here but `sep` below. Why is that?

Reply:
In this path, it was already normalized by the line number 9 of this file to always be `/`.

Reply:
So should we update line 27 and 28 to use `/` instead?

Also what about line 37?

Reply:
27 and 28 run against the output of realpath which is then sep-based.

For inputs into file functions, `/` and `\\` can be used interchangeably.

It's worth noting too that `require('.\\asdf')` isn't even supported in Node.js itself, so users using backslashes is already a very rare with spotty support and shouldn't be done.

Reply:
Also, why change from `path.endsWith` to  `path[path.length - 1] ===`? I think the former reads better.

```suggestion
  if (path.endsWith('/')) return;
```